### PR TITLE
Feature/window

### PR DESCRIPTION
**Pull request by Finkage**
Pull Request to merge to develop

### CODE REVIEW

File: CustomerOrganizer/CustomerOrganizer/MainWindow.py [hunk 1]
```diff
@@ -1,7 +1,31 @@
...
     def __init__(self, master = None):
         super().__init__(master)
         self.master = master
-        self.pack()
\ No newline at end of file
+        self.toolBar()
+
+        self.customer = FileIO.FileIO()
+
+    def donothing():
```
Comment (Alindah) [inline CustomerOrganizer/CustomerOrganizer/MainWindow.py]: I know this is a temporary name and it's going to change anyway, but let's practice consistency. Camel case this please.

File: CustomerOrganizer/CustomerOrganizer/MainWindow.py [hunk 2]
```diff
@@ -1,7 +1,31 @@
...
+
+    def toolBar(self):
+        menubar = Menu(self)
+
+        filemenu = Menu(menubar, tearoff = 0)
+        filemenu.add_command(label = "New Customer", command = self.donothing)
+        filemenu.add_command(label = "Open Customer", command = self.donothing)
+        filemenu.add_command(label = "Save Customer", command = self.customer.saveFile)
+        filemenu.add_command(label = "Save Customer as...", command = self.donothing)
+        filemenu.add_command(label = "Close Customer", command = self.donothing)
```
Comment (Alindah) [inline CustomerOrganizer/CustomerOrganizer/MainWindow.py]: Update "donothing" to use camel case here too.

File: CustomerOrganizer/CustomerOrganizer/MainWindow.py [hunk 3]
```diff
@@ -1,7 +1,31 @@
...
+    def donothing():
+        '''do nothing at all'''
+
+    def toolBar(self):
+        menubar = Menu(self)
+
+        filemenu = Menu(menubar, tearoff = 0)
+        filemenu.add_command(label = "New Customer", command = self.donothing)
+        filemenu.add_command(label = "Open Customer", command = self.donothing)
+        filemenu.add_command(label = "Save Customer", command = self.customer.saveFile)
```
Comment (Alindah) [inline CustomerOrganizer/CustomerOrganizer/MainWindow.py]: I'm getting an unhandled exception error here "'_tkinter.tkapp' object has no attribute 'customer'".

Try initializing `self.customer` before calling `self.toolBar()`

File: CustomerOrganizer/CustomerOrganizer/MainWindow.py [hunk 4]
```diff
@@ -1,7 +1,31 @@
...
-class Application(tkinter.Frame):
+class MainWindow(tkinter.Tk):
     def __init__(self, master = None):
         super().__init__(master)
         self.master = master
-        self.pack()
\ No newline at end of file
+        self.toolBar()
+
+        self.customer = FileIO.FileIO()
```
Comment (Alindah) [inline CustomerOrganizer/CustomerOrganizer/MainWindow.py]: Move this above the `self.toolBar()` line. Also getting an "'FileIO' has no attribute 'FileIO'" error. Change to `self.customer = FileIO()` to get it to work.

**Review by Alindah — CHANGES_REQUESTED** [review]

File: CustomerOrganizer/CustomerOrganizer/MainWindow.py (hunk 1 above)
Comment (Alindah) [reply CustomerOrganizer/CustomerOrganizer/MainWindow.py]: Actually I'm noticing all the tkinter methods use underscores. Maybe we should keep consistent with that instead. (So change this to do_nothing instead. Also please update the saveFile function to use save_file while you're at it.) We can totally discuss this if you think that's a bad idea.

Comment (Finkage) [reply CustomerOrganizer/CustomerOrganizer/MainWindow.py]: underscore ftw

File: CustomerOrganizer/CustomerOrganizer/MainWindow.py (hunk 2 above)
Comment (Finkage) [reply CustomerOrganizer/CustomerOrganizer/MainWindow.py]: its really just filler but i get ya

File: CustomerOrganizer/CustomerOrganizer/MainWindow.py (hunk 3 above)
Comment (Finkage) [reply CustomerOrganizer/CustomerOrganizer/MainWindow.py]: that did work, thanks

File: CustomerOrganizer/CustomerOrganizer/MainWindow.py [hunk 5]
```diff
@@ -3,26 +3,25 @@
...
-    def toolBar(self):
+    def tool_bar(self):
         menubar = Menu(self)
 
         filemenu = Menu(menubar, tearoff = 0)
-        filemenu.add_command(label = "New Customer", command = self.donothing)
-        filemenu.add_command(label = "Open Customer", command = self.donothing)
+        filemenu.add_command(label = "New Customer", command = self.do_nothing)
+        filemenu.add_command(label = "Open Customer", command = self.do_nothing)
         filemenu.add_command(label = "Save Customer", command = self.customer.saveFile)
```
Comment (Alindah) [inline CustomerOrganizer/CustomerOrganizer/MainWindow.py]: Please rename saveFile to save_file. Also do the same in FileIO.py

File: CustomerOrganizer/CustomerOrganizer/MainWindow.py (hunk 4 above)
Comment (Finkage) [reply CustomerOrganizer/CustomerOrganizer/MainWindow.py]: yeah somethings up with VS color coding, FileIO is colored as a library and FileIO.FileIO is colored as a class. ill keep it in mind

File: CustomerOrganizer/CustomerOrganizer/MainWindow.py (hunk 5 above)
Comment (Finkage) [reply CustomerOrganizer/CustomerOrganizer/MainWindow.py]: gotcha

**Pull request merged**